(docs)/fr/docs/basic_applications/summarize/page.mdx

```diff
@@ -19,14 +22,14 @@ Sometimes, you can ask ChatGPT to summarize famous works without pasting in any
   />
 </div>
 
-Although you could instead look up a summary, ChatGPT can tailor the content to your needs. Maybe you really like Football, so you could as ChatGPT to
-“Give me a 100 word summary of ‘Romeo and Juliet’ in American football terms.”
+Bien que vous puissiez chercher un résumé ailleurs, ChatGPT peut adapter le contenu à vos besoins. Peut-être aimez-vous beaucoup le football américain, vous pourriez donc demander à ChatGPT de
+"Me donner un résumé de 100 mots de 'Roméo et Juliette' en termes de football américain."
 
-## Working With Long Blocks of Texts: Copy & Paste!
+## Travailler avec de longs blocs de texte : Copier & Coller !
 
-You can ask ChatGPT to summarize or create an outline of the text and TL;DR the whole thing. All you have to do is copy and paste the specific text into the prompt. (You can include up to 2,500 words. If the work is longer than that, you will have to break it into chunks.)
-Ask for a simplified summary of chosen text by writing, “Summarize this for me like I'm 5 years old: [PASTE TEXT HERE]”
-This one is a summary “for a 5-year old” from a section of Wikipedia’s entry on “[quantum entanglement](https://en.wikipedia.org/wiki/Quantum_entanglement#:~:text=vte-,Quantum%20entanglement,-is%20the%20phenomenon)”.
+Vous pouvez demander à ChatGPT de résumer ou de créer un plan du texte et de faire un TL;DR (trop long; pas lu) de l'ensemble. Tout ce que vous avez à faire est de copier et coller le texte spécifique dans l'invite. (Vous pouvez inclure jusqu'à 2 500 mots. Si l'œuvre est plus longue que cela, vous devrez la diviser en morceaux.)
+Demandez un résumé simplifié du texte choisi en écrivant : "Résumez ceci pour moi comme si j'avais 5 ans : [COLLER LE TEXTE ICI]"
+Voici un résumé "pour un enfant de 5 ans" d'une section de l'article Wikipédia sur "[l'intrication quantique](https://en.wikipedia.org/wiki/Quantum_entanglement#:~:text=vte-,Quantum%20entanglement,-is%20the%20phenomenon)".
```

Comment:
use french wikipedia page for the wikipedia link, and remove the unnecessary text highlighting in the url

```suggestion
Voici un résumé "pour un enfant de 5 ans" d'une section de l'article Wikipédia sur "[l'intrication quantique](https://fr.wikipedia.org/wiki/Intrication_quantique)".
```